### PR TITLE
optimized for package installation on fedora

### PR DESCRIPTION
**Pull request by msaju**
optimized for package installation on fedora

### CODE REVIEW

File: Dockerfile_Fedora [hunk 1]
```diff
@@ -3,16 +3,15 @@ FROM fedora:latest
...
 LABEL project="gluster-build"
-LABEL maintainer "sajmoham@redhat.com"
+LABEL maintainer "gluster-users@gluster.org"
 
-RUN dnf -y update && dnf clean all
+RUN dnf -y --setopt tsflags=nodocs install automake autoconf libtool flex bison openssl-devel  \
+    libxml2-devel python-devel libaio-devel libibverbs-devel librdmacm-devel readline-devel \
+    glib2-devel git  userspace-rcu-devel libcmocka-devel libacl-devel fuse-devel rpcgen make \
+    fedora-packager rpmdevtools libuuid-devel firewalld redhat-rpm-config libtirpc-devel \
+    libcurl-devel liburing-devel -y
```
Comment (mykaul) [inline Dockerfile_Fedora]:
```suggestion
    libcurl-devel liburing-devel
```

File: Dockerfile_Fedora [hunk 2]
```diff
@@ -3,16 +3,15 @@ FROM fedora:latest
...
 
-RUN dnf install automake autoconf libtool flex bison openssl-devel  \
-    libxml2-devel python-devel libaio-devel libibverbs-devel      \
-    librdmacm-devel readline-devel lvm2-devel glib2-devel git     \
-    userspace-rcu-devel libcmocka-devel libacl-devel sqlite-devel \
-    fuse-devel redhat-rpm-config rpcgen libtirpc-devel make \
-    fedora-packager rpmdevtools libuuid-devel firewalld liburing libcurl-devel -y
+RUN yum -y update --setopt tsflags=nodocs && yum clean all && rm -rf /var/cache/yum
```
Comment (mykaul) [inline Dockerfile_Fedora]:
```suggestion
RUN dnf -y update --setopt tsflags=nodocs && yum clean all && rm -rf /var/cache/yum
```

File: Dockerfile_Fedora [hunk 3]
```diff
@@ -3,16 +3,15 @@ FROM fedora:latest
 
 
 LABEL project="gluster-build"
-LABEL maintainer "sajmoham@redhat.com"
+LABEL maintainer "gluster-users@gluster.org"
 
-RUN dnf -y update && dnf clean all
+RUN dnf -y --setopt tsflags=nodocs install automake autoconf libtool flex bison openssl-devel  \
+    libxml2-devel python-devel libaio-devel libibverbs-devel librdmacm-devel readline-devel \
```
Comment (mykaul) [inline Dockerfile_Fedora]: libibverbs-devel , ibcmocka-devel - are not needed.

Comment (msaju) [reply Dockerfile_Fedora]: oh my bad !!!!

File: Dockerfile_Fedora [hunk 4]
```diff
@@ -3,16 +3,15 @@ FROM fedora:latest
 
 
 LABEL project="gluster-build"
-LABEL maintainer "sajmoham@redhat.com"
+LABEL maintainer "gluster-users@gluster.org"
 
-RUN dnf -y update && dnf clean all
+RUN dnf -y --setopt tsflags=nodocs install automake autoconf libtool flex bison openssl-devel  \
+    libxml2-devel python-devel libaio-devel librdmacm-devel readline-devel \
```
Comment (mykaul) [inline Dockerfile_Fedora]: Are you sure we need librdmacm-devel ? I build on Fedora without it.

Comment (msaju) [reply Dockerfile_Fedora]: yes not required, verified it...

**Pull request merged**